ravenpy/models/rv.py

```diff
@@ -712,6 +712,90 @@ def txt_basin_state(self):
         return "\n".join(txt).replace("[", "").replace("]", "")
 
 
+class RVH(RV):
+    def __init__(self, importer, **kwargs):
```

Comment:
This is a *smart* RVH class, in the sense that the data import logic is embedded into it. This is different from the other RV subclasses, whose only logic is how to convert basic python objects to RV content (the smart logic is upstream, mostly in the `base.Raven` class. Some of the *smart* logic at the moment lives inside the `Raven` class, but I suspect that we'll need to rethink this. At the moment, the `Raven` class accepts `latitude`, `longitude`, `area`, scalar arguments, but this might later clash with the distributed configuration. 

I think we should merge this as is for now. 

Once RavenPy becomes a dependency of raven, then maybe we can start thinking about a refactoring. Maybe think about a `Watershed` class, with a `Distributed` subclass (which could be instantiated with the `importer` instance), and a `Global` subclass (with a simple instantiation similar to what is done right now within the `Raven` class). `Raven` would require as an input an instance of `Watershed`.